src/scraper.py

```diff
@@ -0,0 +1,194 @@
+"""Scrapper part1."""
+from __future__ import unicode_literals
+from bs4 import BeautifulSoup
+import io
+import sys
+import re
+import requests
+import geocoder
+import json
+
+
+INSPECTION_DOMAIN = 'http://info.kingcounty.gov'
+INSPECTION_PATH = '/health/ehs/foodsafety/inspections/Results.aspx'
+INSPECTION_PARAMS = {
+    'Output': 'W',
+    'Business_Name': '',
+    'Business_Address': '',
+    'Longitude': '',
+    'Latitude': '',
+    'City': 'Seattle',
+    'Zip_Code': '98101',
+    'Inspection_Type': 'All',
+    'Inspection_Start': '',
+    'Inspection_End': '',
+    'Inspection_Closed_Business': 'A',
+    'Violation_Points': '',
+    'Violation_Red_Points': '',
+    'Violation_Descr': '',
+    'Fuzzy_Search': 'N',
+    'Sort': 'B'
+}
+
+
+def get_inspection_page(**kwargs):
+    """Retrieve the search parameters data from web."""
+    url = INSPECTION_DOMAIN + INSPECTION_PATH
+    params = INSPECTION_PARAMS.copy()
+    for key, val in kwargs.items():
+        if key in INSPECTION_PARAMS:
+            params[key] = val
+    resp = requests.get(url, params=params)
+    resp.raise_for_status()  # <- This is a no-op if there is no HTTP error
+    return resp.content, resp.encoding
+
+
+def write_file(content, name):
+    """Write the reponse to file."""
+    file = open(name, "w")
+    file.write(content)
```

Comment:
If you're not using `with open(filename, mode) as fh:` syntax, you need to make sure to `file.close()` when you're finished with it.